features/org.wso2.carbon.identity.scim2.common.feature/resources/org.wso2.carbon.identity.scim2.common.feature.default.json

```diff
@@ -0,0 +1,10 @@
+{
+  "scim2.enable_schema_extension": true,
+  "scim2.max_bulk_operations": "1000",
+  "scim2.max_bulk_payload": "1048576",
+  "scim2.documentation_uri": "https://docs.wso2.com/display/IS570/apidocs/SCIM2-endpoints/index.html",
```

Comment:
should be 5.9.0 ?

Reply:
The latest document is for 5.7. For 5.8 and 5.9 we do not have this.